packages/admin/admin-theme/src/componentsTheme/MuiFormHelperText.ts

```diff
@@ -13,6 +13,8 @@ export const getMuiFormHelperText: GetMuiComponentTheme<"MuiFormHelperText"> = (
             marginRight: 0,
             color: palette.grey[300],
             marginTop: "4px",
+            display: "flex",
+            gap: spacing(1),
```

Comment:
This styling is specific to the icon rendered in `FieldContainer` correct? 
Then this should probably not be included in the theme-styles of `MuiFormHelperText` but in the `FieldContainer` directly.

Reply:
@kerstin97 I mean both properties, `display` and `gap` 🙂